### PR TITLE
Resolve jPlayer issues

### LINKED ISSUE #51: Incorrect reference to jPlayer
[The enqueue](https://github.com/siteorigin/focus/blob/develop/inc/video.php#L142) for the jPlayer js actually points to the wrong location of /jplayer/jquery.jplayer.(min).js rather than [js/jplayer/jquery.jplayer.(min).js](https://github.com/siteorigin/focus/blob/develop/js/jplayer/jquery.jplayer.min.js).

[Example URL](https://demo.siteorigin.com/focus/2013/02/20/the-mountain/)

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Resolves #51, #52.

### CODE REVIEW

**Comment by Misplon** [comment]
Awesome, thanks man.

Do we not need to update the enqueue?

`wp_enqueue_script('jplayer', get_template_directory_uri().'/jplayer/jquery.jplayer' . SITEORIGIN_THEME_JS_PREFIX . '.js', array('jquery'), '2.9.2');`

**Comment by AlexGStapleton** [comment]
Oh. Crap. Somehow didn't include that with the initial commit. 🤕

File: inc/video.php [hunk 1]
```diff
@@ -131,20 +131,20 @@ function focus_post_video($id = null, $type = null){
 /**
  * Enqueue scripts for the video player.
  */
-function focus_video_enqueue_scripts(){
-	if(is_single()){
+function focus_video_enqueue_scripts() {
+	if( is_single() ){
```
Comment (Misplon) [inline inc/video.php]: @AlexGStapleton We've changed the scope of Focus Videos to include pages too now so we should check this out too.

**Comment by AlexGStapleton** [comment]
There you go.

**Pull request merged**